### PR TITLE
GG-19772 Restore become this user logic.

### PR DESCRIPTION
**Pull request by nva**

### CODE REVIEW

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/security/SecurityConfig.java [hunk 1]
```diff
@@ -77,6 +79,13 @@
     /** Resend activation token. */
     private static final String ACTIVATION_RESEND = "/api/v1/activation/resend";
 
+    /** Switch user url. */
+    private static final String SWITCH_USER_URL = "/api/v1/admin/login/impersonate";
+
+    /** Exit user url. */
+    private static final String EXIT_USER_URL = "/api/v1/logout/impersonate";
+
+
```
Comment (akuznetsov-gridgain) [inline modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/security/SecurityConfig.java]: Not needed empty line.

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java [hunk 2]
```diff
@@ -47,12 +51,14 @@ public UserResponse() {
      * Full constructor.
      *
      * @param acc Account DTO.
+     * @param becameUsed Switch user used flag.
      */
-    public UserResponse(Account acc) {
+    public UserResponse(Account acc, boolean becameUsed) {
```
Comment (smoldachev) [inline modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java]: becameUsed or becomeUsed?

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java [hunk 3]
```diff
@@ -36,6 +36,10 @@
     @ApiModelProperty(value = "Admin flag.")
     private boolean admin;
 
+    /** Switch user used flag. */
+    @ApiModelProperty(value = "Switch user used flag.")
+    private boolean becomeUsed;
```
Comment (smoldachev) [inline modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java]: Why we need this flag?

Comment (nva) [reply modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/model/UserResponse.java]: This flag used in frontend part for change UI

**Pull request merged**